src/yunohost/tests/test_changeurl.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+import time
+import requests
+
+from moulinette.core import init_authenticator
+from yunohost.app import app_install, app_changeurl, app_remove, app_map
+from yunohost.domain import _get_maindomain
+
+from moulinette.core import MoulinetteError
+
+# Instantiate LDAP Authenticator
+AUTH_IDENTIFIER = ('ldap', 'ldap-anonymous')
+AUTH_PARAMETERS = {'uri': 'ldap://localhost:389', 'base_dn': 'dc=yunohost,dc=org'}
+
+auth = init_authenticator(AUTH_IDENTIFIER, AUTH_PARAMETERS)
+
+# Get main domain
+maindomain = _get_maindomain()
+
+
+def setup_function(function):
+    pass
+
+
+def teardown_function(function):
+    app_remove(auth, "change_url_app")
+
+
+def install_changeurl_app(path):
+    app_install(auth, "./tests/apps/change_url_app_ynh",
```

Comment:
change_url script should fix the path before to do operation.
See https://github.com/YunoHost-Apps/wordpress_ynh/blob/master/scripts/.fonctions#L39
 Or it might be done inside the python code